rfc.md

```diff
@@ -322,14 +368,10 @@ This is not a specific design goal of the implementation, but a potentially usef
 
 ## Backward Incompatible Changes
 
-"enum" becomes a language keyword, with the usual potential for naming conflicts with existing global constants.
+"enum" and "type" become language keywords, with the usual potential for naming conflicts with existing global constants.
```

Comment:
Why does type become a keyword?

Reply:
See the section on `match`.  Still not sure that's the correct approach, but it's all I've got for the moment.

Reply:
Oh I forgot about that part. IMO this approach is just too limiting.

Reply:
I am very open to better ideas, but we need some way to indicate that `match()` should do something other than an === check.